plugins/chat/db/migrate/20251119100000_add_pinned_to_user_chat_channel_memberships.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddPinnedToUserChatChannelMemberships < ActiveRecord::Migration[7.1]
+  def change
+    add_column :user_chat_channel_memberships, :pinned, :boolean, default: false, null: false
```

Comment:
Should we create a datetime column instead, like we have for topics? (`pinned_at`)
This would allow sorting the pinned channels based on when the user pinned them, but it would slightly complexify the logic.

Reply:
Discussed internally, we are going to leave the behaviour as is for now.